src/m365/spo/commands/page/page-control-list.ts

```diff
@@ -51,14 +52,16 @@ class SpoPageControlListCommand extends SpoCommand {
         const canvasData: Control[] = clientSidePage.CanvasContent1 ? JSON.parse(clientSidePage.CanvasContent1) : [];
         const controls: any[] = canvasData.filter(c => c.position).map(c => {
           return {
+            id: c.id,
+            type: getControlTypeDisplayName(
+              c.controlType || 0
+            ),
+            title: c.webPartData?.title,
             controlType: c.controlType,
             order: c.position.sectionIndex,
-            id: c.id,
             controlData: {
               ...c
-            },
-            ...c.webPartData || {},
```

Comment:
Rather than removing the whole webPartData object from the result, could we remove just the ID to avoid it overwriting the one on the control?